channel_acceptor.go

```diff
@@ -0,0 +1,111 @@
+package lnd
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// ChannelAcceptor is an interface that represents a predicate on the
+// OpenChannel message and the peer who sent the message.
+type ChannelAcceptor interface {
+	Accept(node *btcec.PublicKey, req *lnwire.OpenChannel) bool
+}
+
+// DefaultChannelAcceptor implements the ChannelAcceptor interface
+// and will be used as the default in the funding manager.
+type DefaultChannelAcceptor struct {
```

Comment:
Is this actually the default? This seems like a special case, RPC controlled variant. 

As far as other default, we may also eventually (in distinct PRs), want to provide a set of acceptors that implement thresholds based off of:
  * mission control state
  * autopilot scoring state (can leverage the scoring system to make decisions off of node connectivity, total channel age, etc)
  * reject nodes we already have channels to
  * etc, etc